plugins/wysiwygarea/plugin.js

```diff
@@ -38,7 +38,7 @@
 					src = '';
 				}
 
-				var iframe = CKEDITOR.dom.element.createFromHtml( '<iframe src="' + src + '" frameBorder="0"></iframe>' );
+				var iframe = CKEDITOR.dom.element.createFromHtml(CKEDITOR.tools.htmlSafeByReview('<iframe src="' + src + '" frameBorder="0"></iframe>', 'Content created inline using safe internal values. url is created using CKEDITOR.tools.fixDomain.'));
```

Comment:
By `url` do you mean `src`? If so, and we're reasonably confident that `fixDomain` won't output something that might escape the attribute value/tag, sounds good!

Reply:
It looks like `fixDomain` is used for IE. The function itself is entirely defined within [tools](https://sourcegraph.com/github.com/ckeditor/ckeditor4/-/blob/core/tools.js?L1385-1415) which looks to be mostly safe. Updated justification from url-> src

Reply:
Ah, you're right, `src` is reset every time it's not IE. Might be worth mentioning this in the justification, but otherwise LGTM.

Reply:
Updated justification